rpcservers/chain/chainnotifier_server.go

```diff
@@ -245,8 +248,9 @@ func (r *ServerShell) CreateSubServer(configRegistry lnrpc.SubServerConfigDispat
 // zero hash.
 //
 // NOTE: This is part of the chainrpc.ChainNotifierService interface.
-func (s *Server) RegisterConfirmationsNtfn(in *ConfRequest,
-	confStream ChainNotifier_RegisterConfirmationsNtfnServer) error {
+func (s *Server) RegisterConfirmationsNtfn(in *chainrpc.ConfRequest,
+	confStream chainrpc.ChainNotifier_RegisterConfirmationsNtfnServer,
```

Comment:
Hmm, this looks very unusual... To be honest, I'd prefer to have a line that's slightly too long than this pattern. The line length isn't enforced by any tool so I guess the implicit "rule" is: try to adhere to the 80 character limit where possible and where it makes sense without affecting readability too much.